homeassistant/components/freedompro/binary_sensor.py

```diff
@@ -0,0 +1,94 @@
+"""Support for Freedompro binary sensors."""
```

Comment:
Please reduce the size of this PR by limiting it to a single platform.

See: <https://developers.home-assistant.io/docs/creating_component_code_review#5-make-your-pull-request-as-small-as-possible>